.changeset/brave-bags-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog': minor
+---
+
+Fixes in kind selectors (now OwnershipCards work again)
```

Comment:
Might want to mention that `EntityKindPicker` now accepts an optional `allowedKinds` prop, which is the actual public-facing change.